CMakeLists.txt

```diff
@@ -313,6 +313,7 @@ endif()
 
 if (UPNP_VERSION_STRING VERSION_GREATER_EQUAL "1.12.0")
     add_definitions(-DUPNP_1_12_LIST)
+    add_definitions(-DUPNP_HAS_IPADDR_AND_OS_IN_FILEINFO)
```

Comment:
This will need to moved to a 1.12.1 test now its released!

Reply:
or should we make a hard depend on 1.12.1?